modules/UI/videolayout/VideoContainer.js

```diff
@@ -497,7 +497,7 @@ export class VideoContainer extends LargeContainer {
 
             const flipX = stream.isLocal() && this.localFlipX && !this.isScreenSharing();
 
-            this.video.style.transform = flipX ? 'scaleX(-1)' : 'none';
+            this.video.style.transform = flipX && !this.isScreenSharing() ? 'scaleX(-1)' : 'none';
```

Comment:
Hum, but `!this.isScreenSharing()` is taken into account two lines above? I don't see the change with this PR?

Reply:
@damencho Sorry for the previous mistake, I just updated the pull request
Thanks in advanced